pkg/server/cloud/aws/instances.go

```diff
@@ -17,7 +17,9 @@ limitations under the License.
 package aws
 
 import (
+	"context"
 	"fmt"
+	"github.com/elotl/kip/pkg/server/nodemanager"
```

Comment:
This is a no-no: even though it's not an explicit circular dependency, the node controller uses the cloud client which is implemented by this package for AWS, so it's pretty close. It's 100x better to hardcode a timeout value than to introduce a circular dependency.

Reply:
good point :+1: I forgot to copy this var to the aws package.